examples/quickstart/hello_world_distributed.cpp

```diff
@@ -47,7 +47,7 @@ std::size_t hello_world_worker(std::size_t desired)
         char const* msg = "hello world from OS-thread {1} on locality {2}\n";
 
         hpx::util::format_to(hpx::cout, msg, desired, hpx::get_locality_id())
-            << hpx::flush;
+            << std::flush;
```

Comment:
Is this something we should enforce more generally (or maybe this is the last use of `hpx::flush`? I doubt it...)? And perhaps mark `hpx::flush` as deprecated?

Reply:
Yes, I think we should deprecate `hpx::flush` and all the other manipulators we have implemented. They are not needed anymore (and have not been for a long time).

Reply:
https://github.com/STEllAR-GROUP/hpx/issues/4736